dts/bindings/mtd/zephyr,target-eeprom.yaml

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+description: Zephyr I2C target EEPROM based on Atmel AT2X
+
+compatible: "zephyr,target-eeprom"
+
+include: ["atmel,at2x-base.yaml", i2c-device.yaml]
```

Comment:
This doesn't make sense to me.  If we are defining a "generic" 'zephyr,target-eeprom' it shouldn't be tied to at2x.

Reply:
I didn't want to break the old compatibility which was also unclear to me, but I did some tests without binding to at2x (which required removing unused properties) and it works fine, so I'm gonna change that.